src/main/java/net/openhft/chronicle/bytes/internal/ChunkedMappedFile.java

```diff
@@ -83,7 +83,7 @@ public static void warmup() {
         try {
             Jvm.setExceptionHandlers(Slf4jExceptionHandler.ERROR, null, null);
 
-            @NotNull final File file = File.createTempFile("delete_warming_up", "me");
+            final File file = File.createTempFile("delete_warming_up", "me", new File("/scratch_area"));
```

Comment:
but will this actually warm up the code in the same way?

Reply:
Yes, I think it will. Merely placing the queue files one level down in the directory structure will likely not change the warmup outcome at all.  I will, however, improve on how the temp dir is created in a new proposal.